force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +347,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+            Integer strPosition;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+            for(String str : words){
+                if(str == wordToFind){
+                    strPosition = words.indexOf(str);
+                    return strPosition;
+                   
+               }
+            }
+            
+        return -1; // Word Not Found.
     }
```

Comment:
In this case I think would be easier to use a traditional for loop with a variable for the index 

Then you can return the index directly. In your case you create a quadratic time solution. When you called indexOf you did a second iteration over the list to to get the index.  DM me or join our DSA sessions if that is not clear